Wikimate.php

```diff
@@ -495,23 +567,87 @@ public function getError()
 /**
  * Models a wiki article page that can have its text altered and retrieved.
  *
- * @author  Robert McLeod
+ * @author  Robert McLeod & Frans P. de Vries
  * @since   December 2010
  */
 class WikiPage
 {
+	/**
+	 * Section list identifier for index
```

Comment:
This wording is not clear to me. What do you mean by "for index"?

Reply:
I just knew that shortcut would boomerang back to me. :wink: Rephrased.

Reply:
Much clearer now, thanks!